internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -346,6 +346,15 @@ func Resource() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"config_server_management_mode": {
```

Comment:
does it need to be optional computed or can be just optional?

Reply:
Exactly, server default is ATLAS_MANAGED